lib/cli-config.js

```diff
@@ -70,12 +72,16 @@ exports.getContent = function(config, directory) {
 
         if (ext === '.js' || ext === '.json') {
             content = require(configPath);
+        } else if (ext === '.yaml') {
+            content = yaml.safeLoad(fs.readFileSync(configPath, 'utf8'));
         } else {
-            content = JSON.parse(
-                stripJSONComments(
-                    fs.readFileSync(configPath, 'utf8')
-                )
-            );
+            data = fs.readFileSync(configPath, 'utf8');
+
+            try {
+                content = JSON.parse(stripJSONComments(data));
+            } catch (e) {
+                content = yaml.safeLoad(data);
```

Comment:
I don't even mind doing it myself :) I just think it would be good in a separate PR, that's all. Appreciate the feedback though, thanks :)

So for now, are you okay with me switching to the code I pasted above? I can update the commit to do that and have the thrown error be a generic `new Error('Parse error in ' + configPath);`.